inputs/templates/terra_workspaces/cohort_mode/cohort_mode_workspace_dashboard.md.tmpl

```diff
@@ -26,7 +26,7 @@ The following cohort-level or batch-level inputs are also required:
 |---------|--------|--------------|
 |`String`|`sample_set_id`|Batch identifier|
 |`String`|`sample_set_set_id`|Cohort identifier|
-|`File`|`cohort_ped_file`|Path to the GCS location of a family structure definitions file in [PED format](https://gatk.broadinstitute.org/hc/en-us/articles/360035531972-PED-Pedigree-format). Sex aneuploidies (detected in `02-EvidenceQC`) should be entered as sex = 0.|
+|`File`|`cohort_ped_file`|Path to the GCS location of a family structure definitions file in [PED format](https://github.com/broadinstitute/gatk-sv#ped-format).|
```

Comment:
Is this a valid link? Would this be better? https://gatk.broadinstitute.org/hc/en-us/articles/360035531972-PED-Pedigree-format

Reply:
This is just an internal link to the relevant section in the README. I tested it on my branch and it worked. As I explained in my PR summary and we discussed in the pipeline meeting, the previous link was not sufficient to describe all the requirements.

Reply:
I had pasted an incorrect link in my previous post, my apologies, I was mainly referring to the gatk-sv website.

You may replace `### PED file format` in `website/docs/gs/input_files.md` with `### PED file format {#ped-format}` and update the link to `https://broadinstitute.github.io/gatk-sv/docs/gs/inputs#ped-format`. This will be linking to the docs instead of README. We will have one less link to worry about not breaking :) when we migrate to the docs website once it is ready.

Reply:
Nothing concerning at this point, so, I will leave it up to you to decide.

Reply:
Thanks for the description of how to add a section link on the website, I was just wondering how to do that! I've updated the website docs with a few more internal links (let me know if you see any issues there). 

Currently all the links on the Terra dashboard are to the README, so I think it makes more sense to keep it consistent for now and update them all at once to point to the website when we're ready.

Reply:
Sounds good, thanks! Sure, we can leave it as-is and we can revisit and update all the links in a consistent way when the website is in a better shape.